cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -88,19 +179,44 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         query("SHOW USERS", assertNodesShown("User", column = "user")) {
           resultTable()
         }
+        p("It is possible to assign multiple roles to multiple users in one command.")
+        query("GRANT ROLES role1, role2 TO user1,user2,user3", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 4)
+        })) {
+          // Can't pull on the result since neither roles nor users exist
+        }
+      }
+      section("Revoking roles from users", "administration-security-roles-revoke") {
+        p("Users can lose access rights by revoking roles from them using `REVOKE ROLE`.")
+        query("REVOKE ROLE myrole FROM jake", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          p("Nothing is returned from this query, except the count of system database changes made.")
+          resultTable()
+        }
+        p("The roles revoked from users can no longer be seen in the list provided by `SHOW USERS`.")
+        query("SHOW USERS", assertNodesShown("User", column = "user")) {
+          resultTable()
+        }
+        p("It is possible to revoke multiple roles from multiple users in one command.")
+        query("REVOKE ROLES role1, role2 FROM user1,user2,user3", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 4)
+        })) {
+          // Can't pull on the result since neither roles nor users exist
+        }
       }
     }
   }.build()
 
   private def assertNodesShown(label: String, propertyKey: String = "name", column: String = "name") = ResultAndDbAssertions((p, db) => {
     val tx = db.beginTransaction(Type.explicit, AnonymousContext.read())
     try {
-      println(p.resultAsString)
+//      println(p.resultAsString)
```

Comment:
Perhaps best to just delete the lines.

Reply:
Did you have a reason for them to be here? There are several in the privilege tests as well - do we want to keep them or remove them? (they do print things when running the tests/building docs)